spec/support/active_record/schema.rb

```diff
@@ -40,6 +40,7 @@
     t.column :created_at, :datetime
     t.column :updated_at, :datetime
     t.column :favourite_device, :string
+    t.column :favourite_colours, :json
```

Comment:
json is only supported on postgres, tests are failing for mysql/sqlite because of this.

Reply:
I ran `bundle exec rspec`. I see now that only runs tests on 1 db.

Will address this by only running the new tests if the db is postgres.

Reply:
@tbrisker The changes you requested have been implemented.